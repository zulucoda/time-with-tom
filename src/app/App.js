import React from 'react';
import styled from 'styled-components';

const Container = styled.div``;

const Header = styled.div`
  background-color: #282c34;
  min-height: 10vh;
  display: flex;
  flex-direction: row;
  font-size: calc(10px + 2vmin);
  color: white;
`;

const Logo = styled.div`
  margin: 20px 0;
  color: #eee;
  font-size: 28px;
  line-height: 0.9;
  font-weight: 200;
  font-family: 'Pacifico';
  text-shadow: 0 1px 1px #000;
  text-align: left;
`;

const Nav = styled.div``;

function App() {
  return (
    <Container>
      <Header>
        <Logo>Time with Tom</Logo>
        <Nav>
          <ul>
            <li></li>
          </ul>
        </Nav>
      </Header>
    </Container>
  );
}

export default App;
