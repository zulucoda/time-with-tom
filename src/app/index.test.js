import { render } from '@testing-library/react';
import App from '.';
import React from 'react';

describe('App - Unit Test', function () {
  test('does it render', () => {
    const { container } = render(<App />);
    expect(container).toMatchSnapshot();
  });
});
