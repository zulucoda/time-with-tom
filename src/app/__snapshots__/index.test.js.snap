// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App - Unit Test does it render 1`] = `
<div>
  <div
    class="sc-fzoiQi gzaSPi"
  >
    <div
      class="sc-AxjAm kmVibe"
    >
      <h1
        class="sc-AxirZ dxeCUY"
      >
        Time with Tom
      </h1>
      <ul
        class="sc-AxiKw TgnyB"
      >
        <li
          class="sc-AxhCb hLIlzG"
        >
          <a
            href="/schedule"
          >
            <span
              class="sc-AxhUy kLxUxn"
            >
              schedule
            </span>
          </a>
        </li>
        <li
          class="sc-AxhCb hLIlzG"
        >
          <a
            href="/booking-request"
          >
            <span
              class="sc-AxhUy kLxUxn"
            >
              booking request
            </span>
          </a>
        </li>
      </ul>
    </div>
    <div
      class="makeStyles-root-1"
    >
      <div
        class="MuiContainer-root MuiContainer-maxWidthMd"
      >
        <div
          class="MuiBox-root MuiBox-root-12"
        >
          <div
            class="MuiPaper-root makeStyles-paper-2 MuiPaper-elevation1 MuiPaper-rounded"
          >
            <h3
              class="MuiTypography-root MuiTypography-h3 MuiTypography-gutterBottom MuiTypography-alignCenter"
              marked="center"
            >
              Schedule
            </h3>
            <table
              class="MuiTable-root WithStyles(ForwardRef(Table))-root-96"
            >
              <thead
                class="MuiTableHead-root WithStyles(ForwardRef(TableHead))-root-99"
              >
                <tr
                  class="MuiTableRow-root WithStyles(ForwardRef(TableRow))-root-101 MuiTableRow-head"
                >
                  <th
                    class="MuiTableCell-root MuiTableCell-head WithStyles(ForwardRef(TableCell))-root-107 WithStyles(ForwardRef(TableCell))-root-108"
                    scope="col"
                  >
                    Name
                  </th>
                  <th
                    class="MuiTableCell-root MuiTableCell-head WithStyles(ForwardRef(TableCell))-root-107 WithStyles(ForwardRef(TableCell))-root-121"
                    scope="col"
                  >
                    Surname
                  </th>
                  <th
                    class="MuiTableCell-root MuiTableCell-head WithStyles(ForwardRef(TableCell))-root-107 WithStyles(ForwardRef(TableCell))-root-122"
                    scope="col"
                  >
                    Email
                  </th>
                  <th
                    class="MuiTableCell-root MuiTableCell-head WithStyles(ForwardRef(TableCell))-root-107 WithStyles(ForwardRef(TableCell))-root-123"
                    scope="col"
                  >
                    Activity
                  </th>
                  <th
                    class="MuiTableCell-root MuiTableCell-head WithStyles(ForwardRef(TableCell))-root-107 WithStyles(ForwardRef(TableCell))-root-124"
                    scope="col"
                  >
                    Date
                  </th>
                  <th
                    class="MuiTableCell-root MuiTableCell-head WithStyles(ForwardRef(TableCell))-root-107 WithStyles(ForwardRef(TableCell))-root-125"
                    scope="col"
                  >
                    Start Time
                  </th>
                  <th
                    class="MuiTableCell-root MuiTableCell-head WithStyles(ForwardRef(TableCell))-root-107 WithStyles(ForwardRef(TableCell))-root-126"
                    scope="col"
                  >
                    End Time
                  </th>
                  <th
                    class="MuiTableCell-root MuiTableCell-head WithStyles(ForwardRef(TableCell))-root-107 WithStyles(ForwardRef(TableCell))-root-127"
                    scope="col"
                  >
                    Paid
                  </th>
                </tr>
              </thead>
              <tbody
                class="MuiTableBody-root WithStyles(ForwardRef(TableBody))-root-128"
              >
                <tr
                  class="MuiTableRow-root WithStyles(ForwardRef(TableRow))-root-130"
                >
                  <td
                    class="MuiTableCell-root MuiTableCell-body WithStyles(ForwardRef(TableCell))-root-131 WithStyles(ForwardRef(TableCell))-root-132"
                  >
                    Muzi
                  </td>
                  <td
                    class="MuiTableCell-root MuiTableCell-body WithStyles(ForwardRef(TableCell))-root-131 WithStyles(ForwardRef(TableCell))-root-133"
                  >
                    Buthelezi
                  </td>
                  <td
                    class="MuiTableCell-root MuiTableCell-body WithStyles(ForwardRef(TableCell))-root-131 WithStyles(ForwardRef(TableCell))-root-134"
                  >
                    muzi@mfbproject.co.za
                  </td>
                  <td
                    class="MuiTableCell-root MuiTableCell-body WithStyles(ForwardRef(TableCell))-root-131 WithStyles(ForwardRef(TableCell))-root-135"
                  >
                    squash
                  </td>
                  <td
                    class="MuiTableCell-root MuiTableCell-body WithStyles(ForwardRef(TableCell))-root-131 WithStyles(ForwardRef(TableCell))-root-136"
                  >
                    2020-04-22
                  </td>
                  <td
                    class="MuiTableCell-root MuiTableCell-body WithStyles(ForwardRef(TableCell))-root-131 WithStyles(ForwardRef(TableCell))-root-137"
                  >
                    09:00
                  </td>
                  <td
                    class="MuiTableCell-root MuiTableCell-body WithStyles(ForwardRef(TableCell))-root-131 WithStyles(ForwardRef(TableCell))-root-138"
                  >
                    13:00
                  </td>
                  <td
                    class="MuiTableCell-root MuiTableCell-body WithStyles(ForwardRef(TableCell))-root-131 WithStyles(ForwardRef(TableCell))-root-139"
                  >
                    yes
                  </td>
                </tr>
                <tr
                  class="MuiTableRow-root WithStyles(ForwardRef(TableRow))-root-130"
                >
                  <td
                    class="MuiTableCell-root MuiTableCell-body WithStyles(ForwardRef(TableCell))-root-131 WithStyles(ForwardRef(TableCell))-root-140"
                  >
                    Muzi
                  </td>
                  <td
                    class="MuiTableCell-root MuiTableCell-body WithStyles(ForwardRef(TableCell))-root-131 WithStyles(ForwardRef(TableCell))-root-141"
                  >
                    Buthelezi
                  </td>
                  <td
                    class="MuiTableCell-root MuiTableCell-body WithStyles(ForwardRef(TableCell))-root-131 WithStyles(ForwardRef(TableCell))-root-142"
                  >
                    muzi@mfbproject.co.za
                  </td>
                  <td
                    class="MuiTableCell-root MuiTableCell-body WithStyles(ForwardRef(TableCell))-root-131 WithStyles(ForwardRef(TableCell))-root-143"
                  >
                    squash
                  </td>
                  <td
                    class="MuiTableCell-root MuiTableCell-body WithStyles(ForwardRef(TableCell))-root-131 WithStyles(ForwardRef(TableCell))-root-144"
                  >
                    2020-04-23
                  </td>
                  <td
                    class="MuiTableCell-root MuiTableCell-body WithStyles(ForwardRef(TableCell))-root-131 WithStyles(ForwardRef(TableCell))-root-145"
                  >
                    09:00
                  </td>
                  <td
                    class="MuiTableCell-root MuiTableCell-body WithStyles(ForwardRef(TableCell))-root-131 WithStyles(ForwardRef(TableCell))-root-146"
                  >
                    13:00
                  </td>
                  <td
                    class="MuiTableCell-root MuiTableCell-body WithStyles(ForwardRef(TableCell))-root-131 WithStyles(ForwardRef(TableCell))-root-147"
                  >
                    yes
                  </td>
                </tr>
                <tr
                  class="MuiTableRow-root WithStyles(ForwardRef(TableRow))-root-130"
                >
                  <td
                    class="MuiTableCell-root MuiTableCell-body WithStyles(ForwardRef(TableCell))-root-131 WithStyles(ForwardRef(TableCell))-root-148"
                  >
                    Muzi
                  </td>
                  <td
                    class="MuiTableCell-root MuiTableCell-body WithStyles(ForwardRef(TableCell))-root-131 WithStyles(ForwardRef(TableCell))-root-149"
                  >
                    Buthelezi
                  </td>
                  <td
                    class="MuiTableCell-root MuiTableCell-body WithStyles(ForwardRef(TableCell))-root-131 WithStyles(ForwardRef(TableCell))-root-150"
                  >
                    muzi@mfbproject.co.za
                  </td>
                  <td
                    class="MuiTableCell-root MuiTableCell-body WithStyles(ForwardRef(TableCell))-root-131 WithStyles(ForwardRef(TableCell))-root-151"
                  >
                    squash
                  </td>
                  <td
                    class="MuiTableCell-root MuiTableCell-body WithStyles(ForwardRef(TableCell))-root-131 WithStyles(ForwardRef(TableCell))-root-152"
                  >
                    2020-04-24
                  </td>
                  <td
                    class="MuiTableCell-root MuiTableCell-body WithStyles(ForwardRef(TableCell))-root-131 WithStyles(ForwardRef(TableCell))-root-153"
                  >
                    09:00
                  </td>
                  <td
                    class="MuiTableCell-root MuiTableCell-body WithStyles(ForwardRef(TableCell))-root-131 WithStyles(ForwardRef(TableCell))-root-154"
                  >
                    13:00
                  </td>
                  <td
                    class="MuiTableCell-root MuiTableCell-body WithStyles(ForwardRef(TableCell))-root-131 WithStyles(ForwardRef(TableCell))-root-155"
                  >
                    yes
                  </td>
                </tr>
                <tr
                  class="MuiTableRow-root WithStyles(ForwardRef(TableRow))-root-130"
                >
                  <td
                    class="MuiTableCell-root MuiTableCell-body WithStyles(ForwardRef(TableCell))-root-131 WithStyles(ForwardRef(TableCell))-root-156"
                  >
                    Muzi
                  </td>
                  <td
                    class="MuiTableCell-root MuiTableCell-body WithStyles(ForwardRef(TableCell))-root-131 WithStyles(ForwardRef(TableCell))-root-157"
                  >
                    Buthelezi
                  </td>
                  <td
                    class="MuiTableCell-root MuiTableCell-body WithStyles(ForwardRef(TableCell))-root-131 WithStyles(ForwardRef(TableCell))-root-158"
                  >
                    muzi@mfbproject.co.za
                  </td>
                  <td
                    class="MuiTableCell-root MuiTableCell-body WithStyles(ForwardRef(TableCell))-root-131 WithStyles(ForwardRef(TableCell))-root-159"
                  >
                    squash
                  </td>
                  <td
                    class="MuiTableCell-root MuiTableCell-body WithStyles(ForwardRef(TableCell))-root-131 WithStyles(ForwardRef(TableCell))-root-160"
                  >
                    2020-04-25
                  </td>
                  <td
                    class="MuiTableCell-root MuiTableCell-body WithStyles(ForwardRef(TableCell))-root-131 WithStyles(ForwardRef(TableCell))-root-161"
                  >
                    09:00
                  </td>
                  <td
                    class="MuiTableCell-root MuiTableCell-body WithStyles(ForwardRef(TableCell))-root-131 WithStyles(ForwardRef(TableCell))-root-162"
                  >
                    13:00
                  </td>
                  <td
                    class="MuiTableCell-root MuiTableCell-body WithStyles(ForwardRef(TableCell))-root-131 WithStyles(ForwardRef(TableCell))-root-163"
                  >
                    yes
                  </td>
                </tr>
                <tr
                  class="MuiTableRow-root WithStyles(ForwardRef(TableRow))-root-130"
                >
                  <td
                    class="MuiTableCell-root MuiTableCell-body WithStyles(ForwardRef(TableCell))-root-131 WithStyles(ForwardRef(TableCell))-root-164"
                  >
                    Muzi
                  </td>
                  <td
                    class="MuiTableCell-root MuiTableCell-body WithStyles(ForwardRef(TableCell))-root-131 WithStyles(ForwardRef(TableCell))-root-165"
                  >
                    Buthelezi
                  </td>
                  <td
                    class="MuiTableCell-root MuiTableCell-body WithStyles(ForwardRef(TableCell))-root-131 WithStyles(ForwardRef(TableCell))-root-166"
                  >
                    muzi@mfbproject.co.za
                  </td>
                  <td
                    class="MuiTableCell-root MuiTableCell-body WithStyles(ForwardRef(TableCell))-root-131 WithStyles(ForwardRef(TableCell))-root-167"
                  >
                    squash
                  </td>
                  <td
                    class="MuiTableCell-root MuiTableCell-body WithStyles(ForwardRef(TableCell))-root-131 WithStyles(ForwardRef(TableCell))-root-168"
                  >
                    2020-04-26
                  </td>
                  <td
                    class="MuiTableCell-root MuiTableCell-body WithStyles(ForwardRef(TableCell))-root-131 WithStyles(ForwardRef(TableCell))-root-169"
                  >
                    09:00
                  </td>
                  <td
                    class="MuiTableCell-root MuiTableCell-body WithStyles(ForwardRef(TableCell))-root-131 WithStyles(ForwardRef(TableCell))-root-170"
                  >
                    13:00
                  </td>
                  <td
                    class="MuiTableCell-root MuiTableCell-body WithStyles(ForwardRef(TableCell))-root-131 WithStyles(ForwardRef(TableCell))-root-171"
                  >
                    yes
                  </td>
                </tr>
              </tbody>
            </table>
          </div>
        </div>
      </div>
    </div>
    <div
      class="sc-fzoyAV izhqEK"
    >
      <div
        class="sc-fzoLag jTmXOx"
      >
        <a
          class="sc-fznxsB letIOy"
          href="https://mfbproject.co.za"
          target="_blank"
        >
          <img
            class="sc-fznWqX jrUNsZ"
            src="/assets/images/mfbproject.png"
          />
          <span
            class="sc-fznJRM dxqqBy"
          >
            created by mfb project
          </span>
        </a>
      </div>
      <div
        class="sc-fzoXzr brsGSn"
      >
        <span
          class="sc-fzpjYC kWGqOa"
        >
          Copyright © 2020 Muzikayise Flynn Buthelezi (zuluCoda)
        </span>
      </div>
    </div>
  </div>
</div>
`;
